calm_adapter/calm_adapter/src/test/scala/uk/ac/wellcome/calm_adapter/CalmXmlResponseTest.scala

```diff
@@ -57,11 +57,11 @@ class CalmXmlResponseTest extends FunSpec with Matchers {
     }
   }
 
-  describe("CALM suummary response") {
+  describe("Calm suummary response") {
```

Comment:
nit, while you're fixing it:

```suggestion
  describe("Calm summary response") {
```